### ISSUE TITLE
Weights??

### ISSUE THREAD

**Issue by RitwickJoshi** [Samarth-991/Yolov5-Tensorflow#2]
is the model using weights of previous weights??

**Comment by RitwickJoshi** [comment]
If yes then where is it exactly?

**Comment by Samarth-991** [comment]
Yes , go to the darknet website , there you will find models yolov4_137

**Comment by RitwickJoshi** [comment]
Okay but the code doesn't seem to use weights at all. 
Can you please specify which file the model is loading the previous weights? 
My last question is which yolov5 model is this a replica of?

**Comment by Samarth-991** [comment]
!wget https://github.com/AlexeyAB/darknet/releases/download/darknet_yolo_v3_optimal/yolov4.conv.137 -- use these weigths
